packages/ui/styles/core.less

```diff
@@ -39,17 +39,11 @@
 	// Selection
 	.enact-unselectable {
 		cursor: default;
-		-ms-user-select: none;
-		-webkit-user-select: none;
```

Comment:
According to https://caniuse.com/?search=user-select, we need `-webkit-` prefix to use in Safari.

Reply:
@MikyungKim hawkeye~

Reply:
OK